starter_code/views/partials/beerCard.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="beerCard">
+    <img src="{{ image_url }}" alt="{{ name }}" />
+    <h2>{{ name }}</h2>
+    <p class="tagline">{{ tagline }}</p>
+    <p class="description">{{ description }}</p>
+</div>
```

Comment:
par exemple ici tu peux rappeler le this (pas obligatoire ici mais pour d'autre projet, ça sera nécessaire) : 
<h2>{{ this.name }}</h2>
    <p class="tagline">{{ this.tagline }}</p>
    <p class="description">{{ this.description }}</p>